assignments/27-02-25/oop/inheritance/Family.py

```diff
@@ -0,0 +1,36 @@
+class Grandfather:
+    def __init__(self, name):
+        self.__name = name
+
+    def __private_method(self):
+        print("This is a private method.")
+
+    def public_method(self):
+        print(f"The family member's name is {self.__name}")
+        self.__private_method()
+
+Grandfather = Grandfather("Ashraf", "5'7")
+Grandfather.display()
+
+class Father(Grandfather):
+    def __init__(self, name, hair):
+        super().__init__(name)
+        self.hair = hair
+
+
+def display(self):
+    self.public_method()
+
+
+Father = Father("Abul", "5'6", "Black")
```

Comment:
- It has only two properties 
`class Father(Grandfather):
    def __init__(self, name, hair)`

- object name should be in lowercase `father = Father("Abul", "5'6", "Black")`